coverage/html.py

```diff
@@ -367,6 +369,11 @@ def write_html_file(self, ftr: FileToReport, prev_html: str, next_html: str) ->
 
         # Write the HTML page for this file.
         file_data = self.datagen.data_for_file(ftr.fr, ftr.analysis)
+
+        contexts=Counter(c for cline in file_data.lines for c in cline.contexts)
```

Comment:
Suprisingly, my linters don't complain about the lack of spaces around the `=` here...